### PR TITLE
Add markdown languages

### LINKED ISSUE #9: Add Internationalization for markdown pages
All pages generated by markdown do not have any internationalization to them. I'm thinking the following

Create separate language specific markdown files with a new language frontmatter property we can add to our graphql queries. Then  make sure gatsby node builds the new language specific pages. Finally confirm that the links go to the language specific markdown file

### PR DESCRIPTION
**Pull request by BruceRW3**
This PR does the following:

- Create a markdown file for every language
- Add types to the useLocale hook and speaker, musician, and book pages
- Add locale to frontmatter

Closes #9

### CODE REVIEW

**Review by jordanpowell88 — CHANGES_REQUESTED** [review]
Let's add some e2e automation around this work

File: .gitignore [hunk 1]
```diff
@@ -45,3 +45,11 @@ Thumbs.db
 /apps/conference/node_modules
 /apps/conference/public
 /apps/conference/.cache
+apps/global/src/types.ts
+apps/global/src/markdown-pages/speakers/ashley-powell.es.md
+apps/global/src/markdown-pages/speakers/ben-swarts.es.md
+apps/global/src/markdown-pages/speakers/hanna-swartz.es.md
```
Comment (jordanpowell88) [inline .gitignore:51]: We don't want these to be ignored

File: apps/global/src/markdown-pages/speakers/hanna-swatz.pt.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
```
Comment (jordanpowell88) [inline apps/global/src/markdown-pages/speakers/hanna-swatz.pt.md:1]: This file name needs renamed. It has a typo

**Review by jordanpowell88 — CHANGES_REQUESTED** [review]
This looks awesome! Just 2 minor changes

File: apps/global/src/markdown-pages/speakers/hanna-swatz.pt.md (hunk 2 above)
Comment (BruceRW3) [reply apps/global/src/markdown-pages/speakers/hanna-swatz.pt.md]: Pulled requested files out of .gitignore.
Corrected typo on file name.

**Review by jordanpowell88 — APPROVED** [review]
Amazing job with this @BruceRW3 !

**Pull request merged**